settings.gradle

```diff
@@ -1,6 +1,6 @@
 pluginManagement {
     repositories {
-        jcenter()
+        mavenCentral()
```

Comment:
I'm unsure why, can you explain?

Reply:
Jcenter [is dead](https://blog.gradle.org/jcenter-shutdown) and Gradle prints a warning if it is used. `mavenCentral` is the recommended replacement, so I went with that.